### PR TITLE
Added aliases and autocomplete.

### PR DESCRIPTION
**Pull request by patmessina**
* Fixed help
* Added documentation
* Started to add more aliases for more resource types

* Added auto complete

Still need to get kfind to search custom made resources. I will do that next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
 # zsh-kuberenetes
 
-This is an oh-my-zsh plugin to make working with kubernetes easier. It provides a bunch of bash aliases and zsh functions.
+This is an oh-my-zsh plugin to make working with kubernetes easier. It provides
+a bunch of bash aliases and zsh functions.
 
 ## Usage
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md]: Link added.

**Comment by patmessina** [comment]
Removed the auto-complete. I found that it is sourcing the oh-my-zsh plugin, and is conflicting with some aliases.

**Comment by Dbz** [comment]
Does this look good for merging to you? After this goes in, I'm going to make a PR to clean up some of my old code that doesn't apply to everyone else.

**Comment by patmessina** [comment]
I think its good to go for now. Im working on the stuff discussed earlier, albeit slowly.

**Pull request merged**